### DISCUSSION TITLE
NTP Server?

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#614] [Q&A]
When is the "NTP Server" field used? The Lay-Z-Spa Module seems to have figured out the current time without my filling in this field.

-Wes

[Image: NTP Server field](http://www.westhaver.com/ntp_server.png)

**Comment by visualapproach** [comment]
When you want to connect to your own ntp

**Comment by dwoosnam** [comment]
Some people will only allow a single source on their network to provide time via NTP. This can provide bandwith reductions and potentially security benefits.

This way only one server will reach out of your network (to the internet) to sync with an external time source, and all your internal machines NTP traffic stay inside your network.

**Reply by visualapproach** [reply]
If field is left empty it will try default NTP servers which is pool.ntp.org and time.nist.gov
When filled in it will try that address before the default.
